sqlitecluster/SQLiteNode.cpp

```diff
@@ -2785,3 +2790,13 @@ void SQLiteNode::kill() {
         peer->reset();
     }
 }
+
+string SQLiteNode::_getLostQuorumLogMessage() const {
+    string lostQuormMessage;
+    if (_lastLostQuorum) {
+        lostQuormMessage = " Lost Quorum at: " + STIMESTAMP_MS(_lastLostQuorum) + " (" + 
+        to_string((double)(STimeNow() - _lastLostQuorum) / 1000000.0) + " seconds ago).";
```

Comment:
Question, why is it valuable to log "seconds ago"? 
Why not just log the UTC timestamp of when it occurred? I feel like with a UTC timestamp we can more easily check the logs to see what happened. Thoughts?

Reply:
We log both, so I don't need to do math in my head to see if this was "recent" or a long time ago. We also have the timestamp if we want to look up the logs at that time.

Reply:
oh i see. cool.